File: TodoApp/TodoApp/Scene/Model/UserModel.swift
//
//  UserModel.swift
//  TodoApp
//
//  Created by Ömer Faruk Öztürk on 20.03.2023.
//

import Foundation

struct UserModel: Codable {
    let id, username, password, mail, nameSurname, profilImage: String?
}

File: Backend/models/UserModel.go
package models

import "go.mongodb.org/mongo-driver/bson/primitive"

type UserModel struct {
	ID          primitive.ObjectID `json:"id,omitempty"`
	Username    string             `json:"username"`
	NameSurname string             `json:"nameSurname"`
	Password    string             `json:"password"`
	Mail        string             `json:"mail"`
	ProfilImage []byte             `json:"profilImage"`
}
